extension/src/popup/components/signTransaction/Operations.tsx

```diff
@@ -1,8 +1,9 @@
 import React from "react";
 import styled from "styled-components";
 
-import { OPERATION_TYPES } from "constants/operationTypes";
+import { OPERATION_TYPES } from "constants/transaction";
 import { COLOR_PALETTE, FONT_WEIGHT } from "popup/constants/styles";
+import { TRANSACTION_WARNING } from "constants/transaction";
```

Comment:
Is this the same source as on line #4?

Reply:
Oh yeah - good catch! Weird that the linter didn't pick that up 🤔